### PR TITLE
front: renaming in lmr

### LINKED ISSUE #9681: lmr: front: a few renamings
# Description

Some namings weren't clear for the users during the tests. Here's a tiny issue to rename some parts of the UI

# AC

- [ ] The option "Dès que possible" in the origin timing has been renamed in "respecter l'horaire de la destination"
- [ ] "Le calcul n'a pas abouti" has been renamed in "Un problème technique est survenu" and the text below has been updated

Image

- [ ] A tab that doesn't find a result now displays "Pas de capactié"

Image

### PR DESCRIPTION
**Pull request by Caracol3**
- To get the 'no capacity' message, try a consist with 30KM/h maxspeed 
- For the 'technical issue' i randomly manage to get it but can't reproduce it now so its hard to test.

close #9681

### CODE REVIEW

File: front/public/locales/fr/stdcm.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
       "downloadSimulationSheet": "Télécharger la fiche de simulation",
       "formatCreationDate": "{{day}}/{{month}}/{{year}} {{hours}}:{{minutes}}",
       "gesicoRequest": "et joignez ce document à votre demande GESICO DSDM",
-      "notFound": "Aucune solution n'a été trouvée pour cette configuration",
+      "notFound": "Aucun passage n'a été trouvé pour cette configuration",
```
Comment (SergeCroise) [inline front/public/locales/fr/stdcm.json:50]:
```suggestion
      "notFound": "Aucun chemin n'a été trouvé pour cette configuration",
```

Comment (SergeCroise) [inline front/public/locales/fr/stdcm.json:49]:
```suggestion
      "notFound": "Aucun chemin n'a été trouvé pour cette configuration",
```
path : passage, chemin ou trajet ?
https://osrd.fr/fr/docs/reference/design-docs/stdcm/pathfinding_module/conflict_detection/

File: front/public/locales/fr/stdcm.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
...
       "conflictsTitle": "Voici les problèmes rencontrés au moment du calcul sur le chemin le plus court :",
```
Comment (SergeCroise) [inline front/public/locales/fr/stdcm.json:50]:
```suggestion
      "conflictsTitle": "Voici les problèmes rencontrés lors du calcul du chemin le plus court :",
```
(feel free to reject or modify my changes)

File: front/public/locales/fr/stdcm.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
...
       "trackConflict": "conflit avec un autre sillon entre <strong>{{waypointBefore}}</strong> et <strong>{{waypointAfter}}</strong> du {{startDate}} à {{startTime}} au {{endDate}} à {{endTime}}",
```
Comment (SergeCroise) [inline front/public/locales/fr/stdcm.json:51]:
```suggestion
      "trackConflict": "conflit avec un autre chemin entre <strong>{{waypointBefore}}</strong> et <strong>{{waypointAfter}}</strong> du {{startDate}} à {{startTime}} au {{endDate}} à {{endTime}}",
```

File: front/public/locales/fr/stdcm.json [hunk 4]
```diff
@@ -47,7 +47,7 @@
...
       "trackConflictSameDay": "conflit avec un sillon entre <strong>{{waypointBefore}}</strong> et <strong>{{waypointAfter}}</strong> le {{startDate}} entre {{startTime}} et {{endTime}}",
```
Comment (SergeCroise) [inline front/public/locales/fr/stdcm.json:52]:
```suggestion
      "trackConflictSameDay": "conflit avec un chemin entre <strong>{{waypointBefore}}</strong> et <strong>{{waypointAfter}}</strong> le {{startDate}} entre {{startTime}} et {{endTime}}",
```
(même remarque que plus haut: chemin ou trajet)

**Comment by SharglutDev** [comment]
Thank you for your proposals, @SergeCroise :)

However, @Caracol3 is just following the text as it is in the mockup (see the last screen in the issue description). The way we use some words in the docs and when we display them to the user can differ regarding the context of where we use them. We can keep that in mind to maybe update the docs later :)

File: front/public/locales/fr/stdcm.json [hunk 5]
```diff
@@ -61,14 +61,14 @@
...
-        "withoutOutputs": "Sans résultat"
+        "withoutOutputs": "Pas de capacité"
       },
       "simulationSelected": "Vous avez retenu cette simulation",
       "startNewQuery": "Démarrez une nouvelle requête",
       "status": {
         "completed": "Calcul terminé",
-        "errorMessage": "Nous sommes désolé de ne pas avoir pu traiter votre demande. L’équipe STDCM a été notifiée et prendra connaissance de la situation prochainement.",
-        "failed": "Le calcul n'a pas abouti"
+        "errorMessage": "Nous sommes désolé de ne pas avoir pu traiter votre demande. L’équipe LMR a été notifiée et prendra connaissance de la situation dès que possible.",
```
Comment (RomainValls) [inline front/public/locales/fr/stdcm.json]: Not sure, but maybe "désolé" should be "désolés" ?

File: front/public/locales/en/stdcm.json [hunk 6]
```diff
@@ -107,6 +107,7 @@
     "origin": "Origin",
     "posteriorPath": "Posterior path",
     "preciseTime": "precise time",
+    "respectDestinationSchedule": "Respect the destination schedule",
```
Comment (SharglutDev) [inline front/public/locales/en/stdcm.json]:
```suggestion
    "respectDestinationSchedule": "respect the destination schedule",
```

File: front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx [hunk 7]
```diff
@@ -52,6 +53,7 @@ const StdcmOpSchedule = ({
   opScheduleTimeType,
   opToleranceValues,
   opId,
+  isOrigin,
```
Comment (SharglutDev) [inline front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx]: This way, you don't have to pass `isOrigin={false}` in `StdcmDestination`

```suggestion
  isOrigin = false,
```

File: front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx [hunk 8]
```diff
@@ -91,6 +91,7 @@ const StdcmOrigin = ({ disabled = false }: StdcmConfigCardProps) => {
         opScheduleTimeType={origin.arrivalType}
         disabled={disabled}
         opId="origin-arrival"
+        isOrigin={true}
```
Comment (SharglutDev) [inline front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx]:
```suggestion
        isOrigin
```

File: front/public/locales/en/stdcm.json (hunk 6 above)
Comment (Caracol3) [reply front/public/locales/en/stdcm.json]: Done

File: front/public/locales/fr/stdcm.json (hunk 5 above)
Comment (Caracol3) [reply front/public/locales/fr/stdcm.json]: Done

Comment (SharglutDev) [reply front/public/locales/fr/stdcm.json]: Oh right, saw it but forgot to make a comment for that ! I agree to.

Capture d’écran 2024-12-05 à 14 26 35
Forgot to send your review ? 😄

File: front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx (hunk 7 above)
Comment (Caracol3) [reply front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx]: Good catch ! Done

File: front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx (hunk 8 above)
Comment (Caracol3) [reply front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx]: Done

File: front/public/locales/fr/stdcm.json (hunk 5 above)
Comment (RomainValls) [reply front/public/locales/fr/stdcm.json]: Hahah, exactly 😂

File: front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx (hunk 7 above)
Comment (SharglutDev) [reply front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx]: Don't forget to remove the props in `StdcmDestination` :)

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, thank you for these changes :)

There is one comment remaining, you can close it after fixing it.

**Review by Yohh — APPROVED** [review]
missclick

**Pull request merged**